MapboxNavigation/UIView.swift

```diff
@@ -70,7 +70,10 @@ extension UIView {
     }
     
     func pinInSuperview(respectingMargins margins: Bool = false) {
-        guard let superview else { return }
+        guard let superview else {
+            assertionFailure("superview was unexpectedly nil")
```

Comment:
I added this while debugging. It seems like a good check, but it's ancillary to my change. I can remove it if anyone is offended.